test/MenuTest.cpp

```diff
@@ -37,6 +37,11 @@ unittest_teardown() {
   enterKey('D');
 }
 
+// 2 goes up in the menu
```

Comment:
I think this comment would be nice to have as a multiline comment associated with the function.

```c++
/**
 * 2 goes up in the menu
 * ...
 */
unittest(MainMenu) {
```

This should help it to show up with function documentation.